a.rb

```diff
@@ -3,4 +3,14 @@ class Error < Exception
 
 raise Error
 
-puts "Hello world"
+def write(message, io=STDOUT)
```

Comment:
Layout/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.